tensorflow/lite/micro/kernels/conv_test.cc

```diff
@@ -215,8 +215,8 @@ TF_LITE_MICRO_TEST(SimpleTestQuantized16x8PerChannel) {
           tflite::testing::kBiasData, bias_quantized, scales, zero_points,
           tflite::testing::kOutputShape, tflite::testing::kGoldenData,
           golden_quantized, output_scale, output_zero_point,
-          &tflite::testing::common_conv_params, tflite::Register_CONV_2D(),
-          output_data));
+          &tflite::testing::common_conv_params,
+          tflite::Register_CONV_2D_INT16X8REF(), output_data));
```

Comment:
leave this as Register_CONV_2D (once it calls into EvalInt16x8)

Reply:
Done